.github/workflows/ci.yml

```diff
@@ -4,24 +4,21 @@ on:
   pull_request:
     branches:
       - main
+  push:
+    branches:
+      - main
```

Comment:
After a pull request is merged, CI will run again on main. It could help in edge cases, like if something is merged without being properly updated based on master.